modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -1066,6 +1069,36 @@ private void _validateListTypeDefinitionId(
 		}
 	}
 
+	private void _validateLocalized(
+			String businessType, boolean localized,
+			ObjectDefinition objectDefinition)
+		throws PortalException {
+
+		if (!FeatureFlagManagerUtil.isEnabled("LPS-146755") || !localized) {
```

Comment:
@MarinhoFeliphe is it possible here the following scenario:The FF is not enabled (so, this method will be returned) and we have the flag localized true?

Reply:
@gabrielwas I've just changed it [here](https://github.com/liferay-objects/liferay-portal/pull/1836/commits/769bc69abb912a45df9422dc3fe8695ca6ed1e25)